scripts/test-all-policy.zeek

```diff
@@ -95,6 +95,8 @@
 @load protocols/conn/known-hosts.zeek
 @load protocols/conn/known-services.zeek
 @load protocols/conn/mac-logging.zeek
+@load protocols/conn/protocol-strings.zeek
+@load protocols/conn/remove-ip-proto-log.zeek
```

Comment:
Another nit, given it's more than just logging now, maybe `disable-unknown-ip-proto-support.zeek`.

I'm thinking of the NEWS entry: To track only supported/known IP protocols, load ``protocols/conn/disable-unknown-ip-protocol-support.zeek`` (or something nicer/shorter.